packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -178,3 +179,19 @@ Dividers can be placed between sections of the card.
 ```ts file='./CardWithDividers.tsx'
 
 ```
+
+## Cards as tiles
+
+Sets of selectable cards may also be used as tiles, and may be made either single selectable or multiselectable by passing the `variant` property to the `selectableActions` object. You may toggle the visibility of the radio or checkbox by additionally passing the `isHidden` property to the `selectableActions` object.
+
+### Single selectable tiles
+
+```ts file='./CardTile.tsx'
+
+```
+
+### Multi selectable tiles
+
```

Comment:
```suggestion
To allow users to select more than 1 tile in a set, set `variant` to "multiple" within the `selectableActions` object of `<CardHeader>`.
```

if this is true? I feel like I'm not seeing this in the example code

Reply:
The default is multiple select so in this case, no variant should be passed.